HousingRepairsOnlineApi.Tests/UseCasesTests/SaveRepairRequestUseCaseTests.cs

```diff
@@ -34,7 +34,7 @@ public SaveRepairRequestUseCaseTests()
         }
 
         [Fact]
-        public async void GivenARepairRequestWithImageARepairIsSaved()
+        public async void GivenARepairRequestWithImageARepairIsSavedWithScheduledStatus()
```

Comment:
I think that the check of scheduled status should be it's own test, not one associated to whether an image is provided or not.